format-specs/schema.json

```diff
@@ -83,31 +85,31 @@
                     "xmin": {
                       "type": "array",
                       "items": [
-                        { "type": "string" },
+                        { "type": "string", "minLength": 1 },
```

Comment:
that doesn't hurt. But isn't that useless given the const value ?

Reply:
The first array element in the schema can be any text.
The second array element is a const value.

The minLength and const are never checking the same value, so no, it's not useless. ;-)

Reply:
oh, right. forgot this was a 2 items array. forgot my comment